test/e2e/helpers.go

```diff
@@ -0,0 +1,35 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	DefaultTimeout  = 5 * time.Second
+	PollingInterval = 250 * time.Millisecond
+)
```

Comment:
Unclear to me at this point what these numbers should be and if we should expose them in the API for users to mess with